echopype/convert/utils/set_groups_ek80.py

```diff
@@ -294,7 +295,7 @@ def set_beam(self, beam_dict):
                 bb = xr.Dataset(
                     {'backscatter_r': (['frequency', 'quadrant', 'ping_time', 'range_bin'], beam_dict['backscatter_r'],
                                        {'long_name': 'Real part of backscatter power',
-                                        'units': 'dB'}),
+                                        'units': 'Hz'}),
```

Comment:
This is backscatter so obviously not in units Hz, but also not in dB. Since these are raw files, perhaps the best is just set it to volts as the WBT measures voltage over a 1 K
 load.